ova/Config_files/wazuh-template.json

```diff
@@ -0,0 +1,1455 @@
+{

```

Comment:
I think this file isn't necessary. The script is downloading it in this line: https://github.com/wazuh/wazuh-packages/blob/df2320802601b207c199c01425748724cca9eb65/ova/Libraries/elastic_functions.sh#L27

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/06b971366820c9d9cbf359ba4c3b01c427772310